packs/data/pathfinder-bestiary.db/goblin-war-chanter.json

```diff
@@ -1712,7 +1712,7 @@
                     "value": 1
                 },
                 "description": {
-                    "value": "<p>The war chanter sings annoying goblin songs, distracting foes with silly and repetitive lyrics. The chanter attempts a @Check[type:performance|traits:concentrate] check against the Will DCs of up to two enemies within @Template[type:emanation|distance:30]{30 feet}. This has the usual traits and restrictions for a Performance check.</p>\n<hr />\n<p><strong>Critical Success</strong> The target takes a -1 status penalty to Perception checks and Will saves for 1 minute.</p>\n<p><strong>Success</strong> As critical success, but the target is affected for only one round.</p>\n<p><strong>Critical Failure</strong> The target is temporarily immune to Goblin Song for 1 hour.</p>\n<hr />\n<p>@Compendium[pf2e.bestiary-effects.Effect: Goblin Song]{Effect: Goblin Song}</p>"
+                    "value": "<p>The war chanter sings annoying goblin songs, distracting foes with silly and repetitive lyrics. The chanter attempts a @Check[type:performance|traits:concentrate] check against the Will DCs of up to two enemies within @Template[type:emanation|distance:30]{30 feet}. This has the usual traits and restrictions for a Performance check.</p>\n<hr />\n<p><strong>Critical Success</strong> The target takes a -1 status penalty to Perception checks and Will saves for 1 minute. @Compendium[pf2e.feat-effects.Goblin Song (Critical Success)]{Goblin Song (Critical Success)}</p>\n<p><strong>Success</strong> As critical success, but the target is affected for only one round. @Compendium[pf2e.feat-effects.Goblin Song (Success)]{Goblin Song (Success)}</p>\n<p><strong>Critical Failure</strong> The target is temporarily immune to Goblin Song for 1 hour.</p>"
```

Comment:
```suggestion
                    "value": "<p>The war chanter sings annoying goblin songs, distracting foes with silly and repetitive lyrics. The chanter attempts a @Check[type:performance|traits:auditory,concentrate,linguistic] check against the Will DCs of up to two enemies within @Template[type:emanation|distance:30]{30 feet}. This has the usual traits and restrictions for a Performance check.</p>\n<hr />\n<p><strong>Critical Success</strong> The target takes a -1 status penalty to Perception checks and Will saves for 1 minute. @Compendium[pf2e.feat-effects.Goblin Song (Critical Success)]{Goblin Song (Critical Success)}</p>\n<p><strong>Success</strong> As critical success, but the target is affected for only one round. @Compendium[pf2e.feat-effects.Goblin Song (Success)]{Goblin Song (Success)}</p>\n<p><strong>Critical Failure</strong> The target is temporarily immune to Goblin Song for 1 hour.</p>"
```
`This has the usual traits and restrictions for a Performance check.`
![image](https://user-images.githubusercontent.com/64166996/176579386-ce1cc643-c92e-49fe-bf19-fa252606451b.png)